webpack.config.js

```diff
@@ -10,6 +10,7 @@ const assetConfig = {
     entry: {
         'checkout-sdk': './src/index.ts',
         'checkout-button': './src/checkout-button.ts',
+        'currency-service': './src/currency-service.ts',
```

Comment:
Do we need to spit out a separate file? I think it's fine exporting it as a part of the main package?